### ISSUE TITLE
Create a "dank" logo for MemeFinder

### ISSUE THREAD

**Issue by Aniq55** [NITDgpOS/MemeFinder#28]

**Comment by khyati18** [comment]
Can I work this on issue ?
Can you please elaborate the logo type.

**Comment by Aniq55** [comment]
@khyati18  sure, go ahead.
Click on the suggested image. Can you draw a vector of that? and add a magnifying glass and a Lit logo on the two hands?

**Comment by khyati18** [comment]
Yes , I will do it soon.

**Comment by khyati18** [comment]
I have done this on the illustator, is this Okay ?

[png.pdf](https://github.com/NITDgpOS/MemeFinder/files/2691751/png.pdf)
![png](https://user-images.githubusercontent.com/43890805/50174290-d8579a00-031f-11e9-9732-431a6bf75a86.png)

**Comment by Aniq55** [comment]
Can you please draw the vector of this? and make him hold a looking glass with one hand.

**Comment by khyati18** [comment]
Is this okay....I will edit it further today ?
And also chage left hand position.

**Comment by khyati18** [comment]
I will edit this further and tell any other specific changes.

**Comment by khyati18** [comment]
I will edit it further , please tell if it is okay.


[New Composition_2018-12-19 10-36-581.pdf](https://github.com/NITDgpOS/MemeFinder/files/2694701/New.Composition_2018-12-19.10-36-581.pdf)

![new composition_2018-12-19 10-36-58](https://user-images.githubusercontent.com/43890805/50218701-d559b980-03b2-11e9-9b11-437709704c0c.png)

**Comment by Aniq55** [comment]
Yes, the general layout seems fine. Also, make it a 2D logo, no need to add shadows and stuff.
